appendices/VK_EXT_wl_drm_lease_display.txt

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018-2021 The Khronos Group Inc.
```

Comment:
So you're welcome to assign copyright to Khronos if you want, but that's not required. All we need is that it be under CC-BY 4.0 and that the CLA is executed on the PR.